gff3tool/lib/inter_model/inter_model.py

```diff
@@ -86,25 +88,31 @@ def check_duplicate(gff, linelist):
         return eSet
 
 def check_incorrectly_split_genes(gff, gff_file, fasta_file, logger):
-    import gff3_to_fasta
-    lib_path = dirname((dirname(abspath(__file__))))
+    import gff3tool.bin.gff3_to_fasta as gff3_to_fasta
+    lib_path = os.path.dirname((os.path.dirname(os.path.abspath(__file__))))
     eCode = 'Emr0002'
     eSet = list()
     gff3_to_fasta.main(gff_file=gff_file, fasta_file=fasta_file, stype='cds', dline='complete', qc=False, output_prefix='tmp', logger=logger)
-    cmd = lib_path + '/ncbi-blast+/bin/makeblastdb'
+    if platform_system != 'Windows':
+        cmd = lib_path + '/ncbi-blast+/bin/makeblastdb'
```

Comment:
Use `os.path.join` will be better pattern here. It handle path of different OSs.

Reply:
Another thing is that based on our experience of **genomics-workspace**, the commands will work even without the `.exe` extension. Therefore, we can get rid of conditional here.